cmd/config_consolidation_test.go

```diff
@@ -131,13 +134,14 @@ func mostFlagSets() []flagSetInit {
 	// getConsolidatedConfig() is used, but they also have differences in their CLI flags :/
 	// sigh... compromises...
 	result := []flagSetInit{}
-	for i, fsi := range []flagSetInit{runCmdFlagSet, archiveCmdFlagSet, cloudCmdFlagSet} {
+	for i, fsi := range []func(globalFlags *commandFlags) *pflag.FlagSet{runCmdFlagSet, archiveCmdFlagSet, cloudCmdFlagSet} {
 		i, fsi := i, fsi // go...
-		result = append(result, func() *pflag.FlagSet {
+		root := newRootCommand(context.Background(), nil, nil)
```

Comment:
hmm this will still do `os.GetEnv()` for some things :confused: so maybe add a short comment in this test file with an explanation that those can't be fully tested yet and a `TODO` to test them in the future, when they can?

Reply:
```suggestion
                // TODO: this still uses os.GetEnv which needs to be removed 
                // before/along adding tests for those fields 
		root := newRootCommand(context.Background(), nil, nil)
```
?

Reply:
Seems good :+1:

Reply:
Erm I think you messed up the indentation and used spaces instead of tabs

Reply:
:facepalm: